### PR TITLE
Favorites

### PR DESCRIPTION
**Pull request by Raj2631**
Will do the functionality today.

### CODE REVIEW

File: client/src/components/Cardv2/Cardtwo.js [hunk 1]
```diff
@@ -20,42 +20,72 @@ import {
...
+    MUTATION_FAVORITE_PROJECT,
+    getVariablesFavorite()
   );
 
   return (
     <Main>
       <CardContainerOutter isApproved={project.isApproved}>
         <button onClick={reactToProject} className='starContainer'>
-          {getAction(project, currentUser) === 'LIKE' ? <Star /> : <StarFill />}
+          {getActionLikes(project, currentUser) === 'LIKE' ? (
```
Comment (ivanms1) [inline client/src/components/Cardv2/Cardtwo.js]: did prettier add the parenthesis?

File: client/src/pages/Favorites/Favorites.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  const { currentUser, loading, error } = useCurrentUser();
+
+  if (loading) {
+    return <Loader />;
+  }
+
+  if (error) {
+    return <p>Please try again later</p>;
+  }
+  console.log(currentUser);
```
Comment (ivanms1) [inline client/src/pages/Favorites/Favorites.js]: remove

File: client/src/pages/Favorites/Favorites.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  return (
+    <Main>
+      <Header />
+      <Container>
+        <Approval>
+          <p>Favorite Projects</p>
+        </Approval>
+        <CardContainer>
+          {currentUser.favoriteProjects.length === 0 ? (
```
Comment (ivanms1) [inline client/src/pages/Favorites/Favorites.js]:
```suggestion
          {!currentUser.favoriteProjects.length ? (
```

File: client/src/components/Cardv2/Cardtwo.js (hunk 1 above)
Comment (Raj2631) [reply client/src/components/Cardv2/Cardtwo.js]: yeah i think, i also changed name though, made a different function for favorite actions so changed names

File: client/src/pages/Favorites/Favorites.js (hunk 2 above)
Comment (Raj2631) [reply client/src/pages/Favorites/Favorites.js]: oh yea

**Review by revengemiroz — APPROVED** [review]

**Review by ivanms1 — APPROVED** [review]

**Pull request merged**